pkg/sentry/loader/loader_arm64.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2018 The gVisor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package loader loads an executable file into a MemoryManager.
+
+// +build arm64
+
+package loader
+
+import (
+	"fmt"
+
+	"gvisor.dev/gvisor/pkg/abi/linux"
+)
+
+// updateArchMemoryManager sets MemoryManager.archMemoryManager.
+func updateArchMemoryManager(args LoadArgs) error {
```

Comment:
In #1978, we decided that vdso_rt_sigreturn isn't arch specific, should we move this code in loader.go?

Reply:
Done.